samples/host_exerciser/host_exerciser_cmd.h

```diff
@@ -175,7 +181,10 @@ class host_exerciser_cmd : public test_command
 	fpgaperf::ptr_t perf(nullptr);
         if (host_exe_->perf_) {
             uid_t uid = getuid();
-            if (uid != 0) {
+            caps = cap_get_file("/usr/bin/host_exerciser");
```

Comment:
Hi Ramya, the file may not always be under /usr/bin/.  Do we need the full path?